### ISSUE TITLE
This plugin gives me a 404 error

### ISSUE THREAD

**Issue by Hund** [gilbitron/Pico-RSS-Plugin#2]
I downloaded the plugin from (git clone) and moved the folder "pico_rss" to my plugins folder "pico/plugins". And then went to visit the adress <url>/feed, but all I get is:

```
Not Found

The requested URL /html/hund/pico/feed was not found on this server.
Apache/2.4.7 (Ubuntu) Server at *** Port 80
```

Did I do something wrong?

**Comment by zznop** [comment]
Ditto

**Comment by theofruitrouge** [comment]
The same problem. I will reinstall pico on a local server to find the problem. Some others plugins are not working on my pico installation too.

**Comment by Sym-O** [comment]
And did you find a solution ? :)
Not exactly the same problem for me. No error message but just loading the home page instead. I'm not using the same server (localhost / php onboard server) so maybe it is related.

**Comment by theofruitrouge** [comment]
Nope. I think there is big problems with pico plugins. At least many plugins does not work correctly with lighttpd for example.

**Comment by Hund** [comment]
My "solution" was to switch to Jekyll. :)

**Comment by Sym-O** [comment]
Very recomforting guys ! Thanks ^^
I'll try to find the solution. I don't want to change CMS.

**Comment by Sym-O** [comment]
Well, I found the solution at least for my problem. I created a new issue since it may not be exactly the same problem as described here. 
see #3 
++ guys.
